action.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+require 'pdf.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Are we expecting some other request method also?
If not we don't need this condition.